tests/common.go

```diff
@@ -3494,75 +3504,146 @@ func DeleteSchedule(backupScheduleName string, clusterName string, orgID string,
 	return nil
 }
 
-// CreateSourceAndDestClusters creates source and destination cluster
+// CreateApplicationClusters adds application clusters to backup
 // 1st cluster in KUBECONFIGS ENV var is source cluster while
 // 2nd cluster is destination cluster
-func CreateSourceAndDestClusters(orgID string, cloudName string, uid string, ctx context1.Context) error {
-	// TODO: Add support for adding multiple clusters from
-	// comma separated list of kubeconfig files
+func CreateApplicationClusters(orgID string, cloudName string, uid string, ctx context1.Context) error {
+	var clusterCredName string
+	var clusterCredUid string
 	kubeconfigs := os.Getenv("KUBECONFIGS")
 	dash.VerifyFatal(kubeconfigs != "", true, "Getting KUBECONFIGS Environment variable")
 	kubeconfigList := strings.Split(kubeconfigs, ",")
 	// Validate user has provided at least 2 kubeconfigs for source and destination cluster
-	if len(kubeconfigList) != 2 {
-		return fmt.Errorf("2 kubeconfigs are required for source and destination cluster")
+	if len(kubeconfigList) < 2 {
+		return fmt.Errorf("minimum 2 kubeconfigs are required for source and destination cluster")
 	}
 	err := dumpKubeConfigs(configMapName, kubeconfigList)
 	if err != nil {
 		return err
 	}
-	// Register source cluster with backup driver
 	log.InfoD("Create cluster [%s] in org [%s]", SourceClusterName, orgID)
 	srcClusterConfigPath, err := GetSourceClusterConfigPath()
 	if err != nil {
 		return err
 	}
 	log.Infof("Save cluster %s kubeconfig to %s", SourceClusterName, srcClusterConfigPath)
-	sourceClusterStatus := func() (interface{}, bool, error) {
-		err = CreateCluster(SourceClusterName, srcClusterConfigPath, orgID, cloudName, uid, ctx)
-		if err != nil && !strings.Contains(err.Error(), "already exists with status: Online") {
-			return "", true, err
+
+	log.InfoD("Create cluster [%s] in org [%s]", destinationClusterName, orgID)
+	dstClusterConfigPath, err := GetDestinationClusterConfigPath()
+	if err != nil {
+		return err
+	}
+	log.Infof("Save cluster %s kubeconfig to %s", destinationClusterName, dstClusterConfigPath)
+
+	ClusterConfigPathMap[SourceClusterName] = srcClusterConfigPath
+	ClusterConfigPathMap[destinationClusterName] = dstClusterConfigPath
+
+	clusterCreation := func(clusterCredName string, clusterCredUid string, clusterName string) error {
+		clusterStatus := func() (interface{}, bool, error) {
+			err = CreateCluster(clusterName, ClusterConfigPathMap[clusterName], orgID, clusterCredName, clusterCredUid, ctx)
+			if err != nil && !strings.Contains(err.Error(), "already exists with status: Online") {
```

Comment:
[Not a review comment / Opinion] If one suspects that an error occurred due to a cluster with the given cluster name already exists, it would be ideal to perform an inspect and check the status field to determine if it is online, instead of relying solely on verifying the content of error message.

Reply:
@mkoppal-px What do you suggest?
If the cluster exists and
- is online : the error message will have the status Online which we are already verifying
- is not online:  the error message will no thave the status Online but will show if its failed/offline

Reply:
I think what @kphalgun-px is saying is, if we find that the cluster already exists, then lets inspect the cluster with the cluster name and check the status of the cluster in the `if condition` here instead of relying on the error test.

Reply:
Yes correct.
@mkoppal-px Is it required to implement or checking in error message is sufficient as the error message also gete the status from cluster Object only[Had raised a bug to get the correct status in error message]

Reply:
https://portworx.atlassian.net/browse/PA-1343
Will address in this PR